### ISSUE TITLE
Spotify web player doesnt work 

### ISSUE THREAD

**Issue by abhigyan-chatterjee** [focus-browser/browser#290]
Whenever i try to play any songs i get the following message 
![image](https://github.com/pulse-browser/browser/assets/98412450/b956e790-cdb8-49c6-b2bb-f2b85e7e34e0)

**Comment by trickypr** [comment]
Probably similar cause to #191, but needs more investigation

**Comment by abhigyan-chatterjee** [comment]
> Probably similar cause to #191, but needs more investigation

Yes , it is quite similar but while in the original post the songs did not play at all , while here the songs play for 2-3 seconds before this message pops up and the song stops playing , albeit the slider keeps on moving.

I can confirm that this issue seems to be specific to pulse , as i could not replicate this issue on the following browsers

- Brave
- Google Chrome
- Ungoogled Chromium
- Firefox
- Waterfox
- Librefox

All tests have been done on the latest version of windows 11 available on my device (Windows 11 Home Single Language,Version 22H2 ,OS build 22621.2428)

**Comment by trickypr** [comment]
I can't seem to reproduce this on either of my Linux computers, so I am going to guess that there is something misconfiguration with the windows build. Not entirely sure when I will get around to fixing this

**Comment by a0ax** [comment]
I have the same issue (x64IntelWin). I think DRM might be broken in some way.
